client/state/reader/teams/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External Dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_TEAMS_REQUEST,
+	READER_TEAMS_REQUEST_FAILURE,
+	READER_TEAMS_REQUEST_SUCCESS,
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( [], {
+	[ READER_TEAMS_REQUEST_SUCCESS ]: ( state, action ) => [ ...state, ...action.teams ],
+} );
+
+export const isRequesting = createReducer( false, {
+	[ READER_TEAMS_REQUEST ]: () => true,
+	[ READER_TEAMS_REQUEST_FAILURE ]: () => false,
+	[ READER_TEAMS_REQUEST_SUCCESS ]: () => false,
```

Comment:
> much of the isRequesting type values aren't really required to control the display of what's going on in Calypso and if we can live without them we ought to strive for that. a minimal Redux tree is simpler, faster, and clearer in what it's trying to convey.

I agree with the fast part.  I think the tradeoff is in _clearer_.  

Having implied meaning to null, empty list, and a populated one is much less clear IMO (versus explicit `isRequesting`). It also breaks when we start wanting to "refresh" things potentially via user action and display a loading ui state.

That being said: this specific case is one where I don't think that we care about ever refreshing or even showing placeholders so I'll look into dropping it